drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/models/ScenarioGridModelTest.java

```diff
@@ -576,7 +576,7 @@ private void commonValidatePropertyUpdate(int columnIndex, boolean isPropertyTyp
         if (isUnique) {
```

Comment:
@yesamer
Related to the above comment, I think a new test is needed, to be sure that reserved keyword are not used

Reply:
@gitgabrio Done.